models/discriminators.py

```diff
@@ -125,63 +125,52 @@ def __init__(self, img_size, kernel_size, num_filters, lr, w_init, bn_init):
     def build(self, input_size):
         activation = lambda l: tf.nn.leaky_relu(l, alpha=0.2)
 
-        self.conv_1 = Conv2D(filters=self.d_dim, kernel_size=(4, 4), strides=(2, 2), padding='same',
-                             kernel_initializer=self.w_init, use_bias=False
-                             )
+        self.conv_block_1 = ConvBlock(filters=self.d_dim, kernel_size=(4, 4), strides=(2, 2), padding='same',
+                                      w_init=self.w_init, bn_init=self.bn_init, activation=activation)
```

Comment:
There is no batchnorm on the first conv which I why I just did it separately. Should probably change it back